python/src/com/jetbrains/python/psi/stubs/PyClassAttributesIndex.java

```diff
@@ -30,6 +33,20 @@
     return StubIndex.getElements(KEY, name, project, GlobalSearchScope.allScope(project), PyClass.class);
   }
 
+  public static Collection<PyTargetExpression> findAttributes(@NotNull String name, @NotNull Project project, GlobalSearchScope scope) {
```

Comment:
Since this index also contains methods, it would be a good idea to name this method `findClassAttributes`.